cpp/src/community/egonet.cu

```diff
@@ -63,56 +68,106 @@ extract(
 {
   auto v           = csr_view.get_number_of_vertices();
   auto e           = csr_view.get_number_of_edges();
-  auto stream      = handle.get_stream();
+  auto user_stream = handle.get_stream_view();
```

Comment:
Sorry for nitpicking, but isn't this a misnomer? Better be user_stream_view or user_sv or any other name indicating this is a view object?

Reply:
f81a2b4